data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/consumer/impl/UnorderedConsumerVerticle.java

```diff
@@ -32,7 +32,7 @@ public UnorderedConsumerVerticle(Initializer initializer,
   @Override
   void startConsumer(Promise<Void> startPromise) {
     this.consumer.exceptionHandler(this::exceptionHandler);
-    this.consumer.handler(this.recordDispatcher);
+    this.consumer.handler(record -> this.recordDispatcher.dispatch(record));
```

Comment:
before this was a handler (wrapping the apply), now it's "just" a `RecordDispatcher`.  

There is no more needed to switch from the "framework" implicit `handle` invocation, to the explicit `dispatch` ?

Reply:
The reason I removed the `Handler` impl for clarity of the contract: `RecordDispatcher.dispatch` returns a `Future`, if you wanna discard that future is up to you. But having `Handler` implementation of `RecordDispatcher.dispatch` created a kind of "false illusion" that the `RecordDispatcher` is implementing the `Handler` contract "properly"

Reply:
thanks for the reasoning!